kani-driver/src/args/mod.rs

```diff
@@ -40,6 +40,7 @@ where
         .unwrap()
 }
 
+#[allow(dead_code)]
```

Comment:
Why?

Reply:
This function is no longer used, so clippy complains. I thought the function is good to keep to use if we decide to obsolete any options in the future, so that we use the same message wording.